internal/provider/resource_tfe_workspace.go

```diff
@@ -112,9 +116,17 @@ func resourceTFEWorkspace() *schema.Resource {
 
 			"auto_destroy_at": {
 				Type:     schema.TypeString,
+				Computed: true,
 				Optional: true,
 			},
 
+			"auto_destroy_activity_duration": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"auto_destroy_at"},
+				ValidateFunc:  validation.StringMatch(regexp.MustCompile(`^\d{1,5}[dh]$`), "must be 1-5 digits followed by d or h"),
```

Comment:
should be 
```suggestion
				ValidateFunc:  validation.StringMatch(regexp.MustCompile(`^\d{1,4}[dh]$`), "must be 1-4 digits followed by d or h"),
```

Reply:
edge case but `0d` or `0h` seems to be a valid value

Reply:
> edge case but `0d` or `0h` seems to be a valid value but fails when trying to apply

validation is mostly an additional nice-to-have to give the user early feedback. I'm fine with the logic here not needing to 100% match the business logic in the API, since it's an edge case